File: trunk/plugin-scala/src/main/scala/fr/paris/lutece/plugins/scala/bean/artifact/Artifact.scala
package fr.paris.lutece.plugins.scala.bean.artifact

import org.hibernate.validator.constraints.NotEmpty
import fr.paris.lutece.plugins.genericjpa.bean.AbstractBean

class Artifact extends AbstractBean[Int] {

  var _id = 0;
  @NotEmpty
  var _version = "";
  @NotEmpty
  var _groupId = "";
  @NotEmpty
  var _artifactId = "";

  def getId(): Int= {
    _id
  }

  def setId(id: Int): Unit = {
    _id = id
  }

}

File: trunk/plugin-scala/src/main/scala/fr/paris/lutece/plugins/scala/bean/artifact/Artifact_.java
package fr.paris.lutece.plugins.scala.bean.artifact;

import javax.persistence.metamodel.SingularAttribute;
import javax.persistence.metamodel.StaticMetamodel;


@StaticMetamodel( Artifact.class )
class Artifact_
{
    public static volatile SingularAttribute<Artifact, Integer> _id;
    public static volatile SingularAttribute<Artifact, String> _version;
    public static volatile SingularAttribute<Artifact, String> _artifactId;
    public static volatile SingularAttribute<Artifact, String> _groupId;
}